### ISSUE TITLE
undefined after_commit callback

### ISSUE THREAD

**Issue by Tao-Galasse** [state-machines/state_machines-activerecord#85]
It seems like the after_commit callback isn't supported, even if it appears in the doc.
I have to be sure the new state is persisted in database before executing some code, and after_transition doesn't assure me that.
I tried to define a block but I got a `NoMethodError: undefined method 'after_commit'`.

Am I doing it wrong, or is it not implemented with activerecord?

**Comment by rafaelfranca** [comment]
`after_commit` is an Active Record method. This gem doesn't undefine or refedine that method. Can you provide an example application that reproduce the issue?

**Comment by Tao-Galasse** [comment]
Sorry for the late reply.

I misunderstood the doc, I thought it was redefined, as I already saw it in other similar gems. To explain my issue, I was trying to perform a job in an `after_transition` block to send a notification ; but sometimes Redis was faster than postgres and doing `my_record.state` in the job returned the previous value. But I refactored it to add `transition.to` as a param of my job and everything goes well.

Thanks for your help :)
